pubnub-chat-impl/src/commonTest/kotlin/com/pubnub/integration/ChannelIntegrationTest.kt

```diff
@@ -987,6 +989,163 @@ class ChannelIntegrationTest : BaseChatIntegrationTest() {
         statusListener02.close()
         connect.close()
     }
+
+    @Test
+    fun inviteSameUserTwice_shouldHandleGracefully() = runTest {
```

Comment:
Inviting a user serves as a passthrough to the Membership API, which updates existing memberships instead of creating duplicates. Therefore, I believe that combining the invitation of users with fetching the members in a single integration test should be sufficient

Reply:
OK. I will remove it.